tests/scripts/test_task.sh

```diff
@@ -16,8 +16,8 @@ pulp_cli file remote create --name "cli_test_file_remote" \
   --url "https://fixtures.pulpproject.org/file-large/PULP_MANIFEST"
 pulp_cli file repository create --name "cli_test_file_repository" --remote "cli_test_file_remote"
 
-task=$(pulp_cli file repository sync --background --name "cli_test_file_repository" 2>&1 | egrep -o "/pulp/api/v3/tasks/[-[:xdigit:]]*/")
+task=$(pulp_cli file repository sync --background --name "cli_test_file_repository" 2>&1 | grep -E -o "/pulp/api/v3/tasks/[-[:xdigit:]]*/")
 pulp_cli task cancel --href "$task"
-test $(pulp_cli task list --name $sync_task --state canceled | jq -r .count) -eq $(($count + 1))
+test "$(pulp_cli task list --name $sync_task --state canceled | jq -r .count)" -eq $((count + 1))
```

Comment:
Nice find! This was an actual bug i believe?

Reply:
The use of $count? Here was the warning:

```
In ./tests/scripts/test_task.sh line 21:
test $(pulp_cli task list --name $sync_task --state canceled | jq -r .count) -eq $(($count + 1))
     ^-- SC2046: Quote this to prevent word splitting.
                                                                                    ^----^ SC2004: $/${} is unnecessary on arithmetic variables.
```